app/common/uicontrols/programmanagement/views/patient-delete.html

```diff
@@ -0,0 +1,86 @@
+<div class="modal fade" id="deletePatientModal" tabindex="-1" role="dialog"
```

Comment:
*  The form fields are not aligned, please follow the structure from [bootstrap docs](https://getbootstrap.com/docs/3.3/css/#forms-horizontal).
*  The confirm button should be the same regardless of the "Obitou" field.

![screenshot from 2017-09-18 12-13-08](https://user-images.githubusercontent.com/1554715/30538232-569f9a4a-9c6d-11e7-8b73-761f0808ea56.png)

![screenshot from 2017-09-18 12-13-29](https://user-images.githubusercontent.com/1554715/30538245-65141e02-9c6d-11e7-89b2-d11419a6f8c4.png)